automatic/mkdocs-material/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,5 @@
+Update-SessionEnvironment
+
+$version = '0.0.0'
```

Comment:
@stffabi As soon as it is merged AU will run, update the version to the latest available version and commit the changes. Also will push to the community feed.